GitUI/CommandsDialogs/BrowseDialog/GitStatusMonitor.cs

```diff
@@ -386,16 +407,6 @@ private void Update()
 
                             throw;
                         }
-                        finally
-                        {
-                            lock (_statusSequence)
-                            {
-                                if (!ModuleHasChanged() && !cancelToken.IsCancellationRequested)
-                                {
-                                    _commandIsRunning = false;
-                                }
-                            }
-                        }
```

Comment:
Why can this be omitted?
I guess because `CurrentStatus = GitStatusMonitorState.Stopped;` will be set and `_commandIsRunning` is reset when entering the state `GitStatusMonitorState.Running`.

Reply:
See above